src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/YesSqlOptions.cs

```diff
@@ -19,4 +19,6 @@ public class YesSqlOptions
     public IContentSerializer ContentSerializer { get; set; }
 
     public string TablePrefixSeparator { get; set; } = "_";
+
+    public IdentityColumnSize IdentityColumnSize { get; set; } = IdentityColumnSize.Int32;
```

Comment:
I still insist to change the enum name in `YesSQL`, especially if we need to support more types in the future